src/test/resources/features/Curriculum.feature

```diff
@@ -0,0 +1,50 @@
+Feature: creates curriculums
+
+  @DeleteCurriculum
+  Scenario: User create a curriculum with the minimum data required
```

Comment:
Please be more consistent with your other escenarios, maybe you could delete "User" and just have: "Create a curriculum with the minimum data required."